nipst/nipst.go

```diff
@@ -197,19 +192,11 @@ func (nb *NIPSTBuilder) BuildNIPST(challenge *common.Hash) (*types.NIPST, error)
 	if nipst.PostProof == nil {
 		nb.log.Info("starting PoST execution (challenge: %x)", nb.state.PoetProofRef)
 
-		proof, err := nb.postProver.execute(nb.id, nb.state.PoetProofRef, defTimeout)
+		proof, err := nb.postProver.Execute(nb.state.PoetProofRef)
 		if err != nil {
 			return nil, fmt.Errorf("failed to execute PoST: %v", err)
 		}
 
-		res, err := nb.verifyPost(proof, nb.postCfg.SpacePerUnit, nb.postCfg.NumProvenLabels, nb.postCfg.Difficulty)
-		if err != nil {
-			return nil, fmt.Errorf("created an invalid PoST proof: %v", err)
-		}
-		if !res {
-			return nil, fmt.Errorf("created an invalid PoST proof")
-		}
-
 		nb.log.Info("finished PoST execution (proof: %v)", proof)
```

Comment:
not sure if it is a good idea to print the entire post proof..

Reply:
Fixed.